examples/as_promises.js

```diff
@@ -12,10 +12,31 @@ let cardDetails = {
     'postal_code':      10320,
     'number':           '4242424242424242',
     'expiration_month': 2,
-    'expiration_year':  2017,
+    'expiration_year':  2024,
   },
 };
+//Auto capture a charge
```

Comment:
Hey, can we switch to `async/await` in our example code instead of continuing with this deeply nested callback structure? `async/await` makes the code way easier to understand, and  people don't really prefer this callback hell anymore.

Reply:
This will require more time, maybe separate PR & task ?

Reply:
Yes, please, let's do in separate PR/task